### PR TITLE
DXP Cloud Client to Site VPN example

### PR DESCRIPTION
**Pull request by justinchoi001**
This should have neon warning lights flashing that is visible afar.

### CODE REVIEW

File: docs/dxp-cloud/latest/en/infrastructure-and-operations/networking/setting-up-a-client-to-site-vpn-example.md [hunk 1]
```diff
@@ -0,0 +1,270 @@
+# Setting up a Client-to-Site VPN Example
+
+Liferay DXP Cloud team has published the following examples demonstrate to set up an IPsec and also an OpenVPN test server using Ubuntu Server 18.0.4. See the [client to site VPN](./client-to-site-vpn.md) article to learn more about client-to-site VPNs in general.
```
Comment (jrhoun) [inline docs/dxp-cloud/latest/en/infrastructure-and-operations/networking/setting-up-a-client-to-site-vpn-example.md]: 1. How you worded this opening paragraph makes it sound like the author is a separate party from the DXP Cloud team - which is not the case. We're all on the same team. Please reword.

2. `published the following examples demonstrate to set up` - invalid grammar

File: docs/dxp-cloud/latest/en/infrastructure-and-operations/networking/setting-up-a-client-to-site-vpn-example.md [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+
+```warning::
+   Note that example and values may be subject to change and should be adapted for each customer's specific environment.
```
Comment (jrhoun) [inline docs/dxp-cloud/latest/en/infrastructure-and-operations/networking/setting-up-a-client-to-site-vpn-example.md]: Unnecessary to say "Note" when this whole block of text is being "noted" by the admonition.

**Comment by jrhoun** [comment]
Please re-send after resolving my comments and re-reviewing your draft. Thank you.

File: docs/dxp-cloud/latest/en/infrastructure-and-operations/networking/setting-up-a-client-to-site-vpn-example.md (hunk 2 above)
Comment (justinchoi001) [reply docs/dxp-cloud/latest/en/infrastructure-and-operations/networking/setting-up-a-client-to-site-vpn-example.md]: Revised

File: docs/dxp-cloud/latest/en/infrastructure-and-operations/networking/setting-up-a-client-to-site-vpn-example.md (hunk 1 above)
Comment (justinchoi001) [reply docs/dxp-cloud/latest/en/infrastructure-and-operations/networking/setting-up-a-client-to-site-vpn-example.md]: Revised

**Pull request closed without merging**